config/sync/user.mail.yml

```diff
@@ -3,115 +3,28 @@ _core:
 langcode: hy
 cancel_confirm:
   subject: 'Account cancellation request for [user:display-name] at [site:name]'
-  body: |-
```

Comment:
Is it desired intended change? Or just the result of saving the form?

Reply:
Yes, it's the result of saving Account settings form. There are not actual changes only another approach for line endings.

Reply:
Let's exclude this from PR now.